lib/cli.js

```diff
@@ -676,9 +676,9 @@ function getAppStatusTail(repo) {
   var barOptions = {};
   barOptions.last = null;
   barOptions.points = {
-    'creating gear': 1,
+    'creating app': 1,
     'creating DNS alias': 2,
-    'cloning gear and user repositories': 2,
+    'cloning app and user repositories': 2,
```

Comment:
cant change this. these values come from server and assign point value based on time and value of the messgae